lib/class-wp-theme-json-gutenberg.php

```diff
@@ -1178,6 +1178,17 @@ public function get_custom_css() {
 		// Add the global styles root CSS.
 		$stylesheet = $this->theme_json['styles']['css'] ?? '';
 
+		// Add the global styles elements CSS.
+		if ( isset( $this->theme_json['styles']['elements'] ) ) {
+			foreach ( $this->theme_json['styles']['elements'] as $element => $node ) {
+				$custom_element_css = _wp_array_get( $this->theme_json, array( 'styles', 'elements', $element, 'css' ) );
```

Comment:
We're trying to avoid using `_wp_array_get()` whenever possible, so in this case, an `asset()` check would be more appropriate 👍 
```suggestion
				$custom_element_css = isset( $this->theme_json['styles']['elements'][ $element ]['css'] )
					? $this->theme_json['styles']['elements'][ $element ]['css']
					: null;
```